src/encrypter.ts

```diff
@@ -83,9 +83,6 @@ export class Encrypter {
 
       clonedOptions.minPoolSize = 0;
 
-      clonedOptions.promoteValues = false;
```

Comment:
No longer needed as the state machine in mongodb-client-encryption handles adding of these options.